controller/types.go

```diff
@@ -20,77 +20,75 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const (
-	// StatusPassed define the pass value
-	StatusPassed = "Pass"
-	// StatusFailed define the Fail value
-	StatusFailed = "Fail"
-)
-
 var registeredResultMetrics []string
 
 // Declare the fixed chaos metrics. Dynamic (testStatus) metrics are defined in metrics()
 var (
-	ExperimentsTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "c",
+	EngineTotalExperiments = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
If you not use the engine name all where it will be easy to sync it with close-source, Here instead of it you can use TotalExperiments